distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/callables/GetConfigurationProperties.java

```diff
@@ -65,11 +68,18 @@ public GetConfigurationProperties(ConfigurationAdmin configAdmin, String propert
         String query = String.format("(%s=%s)", propertyName, propertyValue);
         Configuration[] configurations = configAdmin.listConfigurations(query);
 
-        if ((configurations == null) || (configurations.length == 0)) {
+        if (configurations == null) {
```

Comment:
There are two persistence managers now and can return the same object. Also, it is just test code, so I wasn't worrying too much about efficiency (it is only used in 2 tests).